### PR TITLE
fix for #3025

### PR DESCRIPTION
**Pull request by D4imyo**
a tool and its dependencies 
take your time :))

### CODE REVIEW

**Comment by noptrix** [comment]
@D4imyo lol :D thank you mate! yeah we need to take time here ...

File: packages/python-srsly/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-srsly
+_pkgname=srsly
+pkgver=2.4.1
+pkgrel=1
+pkgdesc='Modern high-performance serialization utilities for Python.'
+arch=('i686' 'x86_64' 'armv7' 'aarch64')
```
Comment (noptrix) [inline packages/python-srsly/PKGBUILD:9]: we support only 'x86_64' and 'aarch64. the others can be removed. please

File: packages/python-murmurhash/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
```
Comment (noptrix) [inline packages/python-murmurhash/PKGBUILD:1]: no need for combo/split package here, e.g. remove python2 package.please

**Comment by noptrix** [comment]
@D4imyo remove the combo packages please. so remove python2 packages. the rest looks fine so far!

**Comment by D4imyo** [comment]
> @D4imyo remove the combo packages please. so remove python2 packages. the rest looks fine so far!

done.
but why removing them ?

**Review by noptrix — COMMENTED** [review]
> > @D4imyo remove the combo packages please. so remove python2 packages. the rest looks fine so far!
> 
> done.
> but why removing them ?

cause we try to drop python2 support as much as possible. since the deps are not needed by any other packages we skip of combo packages and adding py2 support.

**Comment by noptrix** [comment]
@D4imyo thanks a lot again!

**Pull request merged**